content/millennium/r4/proprietary-codes.md

```diff
@@ -342,6 +342,50 @@ This code set is customized by clients and defines exam rooms, surgery rooms, an
       "display": "Radiology Exam Room",
     }
 
+#### Code Set 18736 Account Types
+
+This code set includes Account Types such as A/R and CASH.
+
+    {
+      "system": "https://fhir.cerner.com/<EHR source id>/codeSet/18736,
+      "code": "18736",
+      "display": "Cash",
+      "userSelected": true
+    }
+
+#### Code Set 18934 Currency Type
+
+This code set includes currency types such as USD.
+
+    {
+      "system": "https://fhir.cerner.com/<EHR source id>/codeSet/18736,
+      "code": "11074",
+      "display": "USD",
+      "userSelected": true
+    }
+
+#### Code Set 24451 Benefit Order Status
+
+This code set includes Functional Balance Account Statuses such as In Process and Ready to bill.
```

Comment:
Kind of confusing the code set is called Benefit Order Status but it includes Functional Balance Account Statuses?

Reply:
So I talked with Adam Keally after talking with Alex. Decided on this:

This code set includes the statuses of a benefit order within the billing workflow such as In Process and Ready to bill.

Reply:
Talked with Kathy, she would like

'of a benefit order'

to be removed

Reply:
https://github.com/cerner/fhir.cerner.com/pull/291/commits/33a2728b5b30cdf435626c1cd9889cd95857ed7d#diff-f1a7812a3d6e9f4049fb6005eecfb1f5R369

+1